docs/UserGuide.md

```diff
@@ -78,19 +115,19 @@ This user guide provides in-depth descriptions of all the available features in
 Note that the app contains **sample data**.<br><br>
    ![Ui](images/Ui.png) <br><br>
 
-5. Type the command in the command box and press _Enter_ to execute it. E.g., typing **`help`** and pressing _Enter_ will open the help window.<br><br>
+5. Type the command in the command box and press Enter to execute it. E.g., typing **`help`** and pressing Enter will open the help window.<br><br>
 
-6. To kickstart your journey, you can add a guest, `John Doe`, to GuestBook using the **`add`** command.
-   * **`add`** `n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3 rq/Extra towels`: Adds a guest named `John Doe` to GuestBook. 
+6. To kickstart your journey, you can add a guest, _John Doe_, to GuestBook using the **`add`** command.
+   * **`add`** `n/John Doe p/98765432 e/johnd@example.com rm/05-73 dr/13/09/22 - 15/09/22 ng/3 rq/Extra towels`: Adds a guest named _John Doe_ to GuestBook. 
 <br><br>
     
-7. To find `John Doe`, you can use the **`find`** command, and search through any of `John Doe`'s `fields`, such as his _name_ or _room_. The command below can help you find guests that match the keywords you entered.
-   * **`find`** `John Doe` : Finds all the guests who have at least one field containing `John` or `Doe` (_case-insensitive_).
+7. To find _John Doe_, you can use the **`find`** command, and search through **any of _John Doe's_ fields**, such as his `name` or `room`. The command below can help you find guests that match the keywords you entered.
+   * **`find`** `John Doe` : Finds all the guests who have at least one field containing _John_ or _Doe_ (case-insensitive).
```

Comment:
Should we add the For example, ... for consistency?

Reply:
Agreed